### DISCUSSION TITLE
Migration to -> Gnome 42

### DISCUSSION THREAD

**Discussion by BinaryGenius** [zb3/gnome-gamma-tool#4] [Ideas]
I'm not the one who wrote this but I would like to help speed up development what is holding the ggt from working on Gnome 42? And are there any workarounds you have in mind but maybe haven't tried yet?

**Comment by zb3** [comment]
Ah, thanks for spotting a mistake in the description :)
It could make people think the tool is outdated and no longer works :(

Yet I actually tested this on F36 with GNOME 42, but updated just the readme and forgot about the description...

Just to make sure I'm not missing anything, can you confirm the tool works for you on GNOME 42? If so, I'll update the description.

**Reply by BinaryGenius** [reply]
I will test it but I don't think I'll be needing ggt anymore I found this extension for gnome and, 1 it's an extension, 2 it has a slider, and 3 it works with night light. There are so many cool features that it completely outclasses ggt, ggt was a nice work around but Soft Brightness is way better and here is the repo https://github.com/F-i-f/soft-brightness. I think what this script can have over this extension is kde support I'm testing out Plasma and I'm using redshift until I find some good brightness control. I see that it's cross DEs already with Cinnamon and Gnome.

**Reply by zb3** [reply]
I think I've seen that one before writing ggt, but while it undoubtedly offers much better user experience for your usecase, it doesn't solve my problem at all since I'd needed to change the **gamma**.

From the developer perspective, that extension is way more complex and it requires way more maintenance and knowledge to modify it, that's why I wrote this from scratch instead of trying to extend existing solutions.

Thanks for helping me make ggt better though! :)

**Reply by BinaryGenius** [reply]
Works on Zorin OS and that's based on ubuntu so I think it's safe to say it works on ubuntu as a whole. And I think I'll use your script in the meantime since the extension has a really annoying [issue](https://github.com/F-i-f/soft-brightness/issues/31) with the mouse that hasn't been solved yet.

**Reply by BinaryGenius** [reply]
Update nevermind Zorin OS is using gnome 3.38.4 wow I didn't realize it was so far behind on updates.

**Reply by BinaryGenius** [reply]
Confirmed works on Fedora 36 I think it works on Fedora 37 Beta too. You can update the description.
